iphone/CoreApi/CoreApi.xcodeproj/project.pbxproj

```diff
@@ -657,7 +665,7 @@
 				COPY_PHASE_STRIP = NO;
 				CURRENT_PROJECT_VERSION = 1;
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
-				DEFINES_MODULE = NO;
+				DEFINES_MODULE = YES;
```

Comment:
Does this PR really require this change? If no, can it be done later separately?

Reply:
Ok!